src/features/create-dao/CreateDAO.types.tsx

```diff
@@ -0,0 +1,89 @@
+import { IconLightning2 } from '@zero-tech/zui/components/Icons';
+import { DropdownIconLabel } from '../ui/DropdownIconLabel/DropdownIconLabel';
+import {
+	Step,
+	ToggleOptions,
+	MediaType,
+	DropdownItem,
+} from '@zero-tech/zui/components';
+
+export const steps: Step[] = [
+	{
+		id: 'details',
+		title: 'Details',
+	},
+	{
+		id: 'governance',
+		title: 'Governance',
+	},
+	{
+		id: 'treasury',
+		title: 'Treasury',
+	},
+	{
+		id: 'launch',
+		title: 'Launch',
+	},
+];
+
+export const VOTING_PROCESS_OPTIONS: ToggleOptions = [
+	{
+		key: 'absolute',
+		label: 'ABSOLUTE',
+	},
+	{
+		key: 'majority',
+		label: 'MAJORITY',
+	},
+];
+
+export const getVotingPeriodItems = (setFieldValue: any): DropdownItem[] =>
```

Comment:
nit: square brackets for these - makes them a little easier to parse. I'll make a task for adding this to our lint rules ☺️